src/controller/mod.rs

```diff
@@ -104,6 +111,24 @@ impl<T> Runtime<T> {
         })
     }
 
+    #[cfg(feature = "containers")]
+    async fn setup_containers(
+        store_dir: &std::path::Path,
+        client: T,
+        tasks: &mut JoinSet<stable_eyre::Result<()>>,
+    ) -> Result<mpsc::Sender<Box<edgehog_containers::requests::ContainerRequest>>, DeviceManagerError>
+    where
+        T: Client + Send + Sync + 'static,
+    {
+        let (container_tx, container_rx) = mpsc::channel(8);
```

Comment:
Why don't we add here a const value, like CONTAINER_CHANNEL_SIZE?

Reply:
I could, but there are many other channels initialized in this file like this, without constants.

I think the channel configurations should be refactored in another PR, maybe an issue should be opened about it if we want to make them configurable. Anyway, I believe it's separate from the current changes.